backend/mobilityAI/Controllers/PatientsController.cs

```diff
@@ -0,0 +1,180 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using Microsoft.AspNetCore.Http;
+using mobilityAI.Models;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Linq;
+using System;
+using System.IO;
+using System.Globalization;
+using Microsoft.VisualBasic;
+using Newtonsoft.Json;
+using System.Text;
+using System.Security.Cryptography;
+
+namespace mobilityAI.Controllers {
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PatientsController : ControllerBase
+    {
+        private readonly MobilityAIContext _context;
+        private static bool isFirstRun = true;
+        private static readonly DateTime epoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        public PatientsController(MobilityAIContext context){
+            _context = context;
+        }
+
+        /// <summary>
+        /// Returns a list of all patients
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        public JsonResult GetPatients() {
+            if (isFirstRun)
+            {
+                var demoPatients = new List<Patient> {
```

Comment:
Can we move this from here to Models/MobilityAIContext and have it run with migrations on db creation instead of like this (see: https://github.com/pgunasekara/mobilityai/blob/master/backend/mobilityAI/Models/MobilityAIContext.cs#L72)?

Reply:
Actually, let's move it later, it's fine to leave as is for this. Can you make a ticket for it?